extension/js/inspector/app/modules/Data.js

```diff
@@ -67,6 +70,7 @@ define([
 
       var modelData = event.data;
       this.modelCollection.add(modelData);
+      this.classCollection.add(modelData);
```

Comment:
two things:
1. do we need all of modelData, or just classId? `_.pick(modelData, 'classId')`